doc/specifications/rest/REST-API-V2.rst

```diff
@@ -6537,6 +6537,164 @@ Example
+Resource                                                          POST                GET                      PATCH/PUT                   DELETE           COPY
+----------------------------------------------------------------- ------------------- ----------------------- ---------------------------- ---------------- --------------
+/permissions/content/objects/<ID>                                 .                   load permissions for a  .                            .                .
+                                                                                      content
+/permissions/content/locations/<path>                             .                   load permissions for a  .                            .                .
+                                                                                      location
+================================================================= =================== ======================= ============================ ================ ==============
+
+
+Specification
+-------------
+
+Managing Permissions
+~~~~~~~~~~~~~~~~~~~~
+
+
+Load Permissions for a Content
+``````````````````````````````
+:Resource: /permissions/content/objects/<ID>
+:Method: GET
+:Description: Loads the permissions for the content object with the given id, for the current user. Depending on the Accept header the content object is embedded.
+:Headers:
+    :Accept:
+        :application/vnd.ez.api.ContentPermissions+xml:  if set the permissions for the content object, with the content object embedded, are returned in xml format (see Content_)
+        :application/vnd.ez.api.ContentPermissions+json:  if set the permissions for the content object, with the content object embedded, are returned in json format (see Content_)
+        :application/vnd.ez.api.ContentPermissionsInfo+xml:  if set the permissions for the content object are returned in xml format (see Content_)
+        :application/vnd.ez.api.ContentPermissionsInfo+json:  if set the permissions for the content object are returned in json format (see Content_)
+    :If-None-Match: <etag> If the provided etag matches the current etag then a 304 Not Modified is returned. The etag changes if the meta data was changed - this happens also if there is a new published version.
+:Parameters:
+    :languages: (comma separated list) restricts the output of translatable fields to the given languages (when embedding the content object)
+:Response:
+
+
+.. code:: http
+
+          HTTP/1.1 200 OK
+          ETag: "<ETag>"
+          Content-Type: <depending on accept header>
+          Content-Length: <length>
+.. parsed-literal::
+Content_
+
+:Error Codes:
+    :401: If the user is not authorized to read the object. This could also happen if there is no published version yet and another user owns a draft of this content
```

Comment:
>    I don't see the point of that

Asking for permissions on an unpublished object is unlikely, but possible, isn't it..?

>    shouldn't this be a 403 ?

If so, then `/content/objects/<ID>` should be changed too... same situation. The 401 situation is possible too I guess, but less likely.